From 7d91170813008da336cdd8df945193fe262d6ff9 Mon Sep 17 00:00:00 2001
From: Alexandre Courbot <acourbot@nvidia.com>
Date: Wed, 11 Jan 2017 17:26:13 +0900
Subject: [PATCH 4/6] [HACK] disable LLVM for cross builds

8c98e27074240a97daadddb0ba3a7e40cc7a646d makes use of the local LLVM
even when cross-compiling. This break the x86/arm cross compilation.
Workaround this for now until I find a proper fix.
---
 configure.ac | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/configure.ac b/configure.ac
index 2e26b8f..3c30a82 100644
--- a/configure.ac
+++ b/configure.ac
@@ -896,13 +896,14 @@ if test "x$have_libelf" = xno; then
    AC_SUBST([LIBELF_CFLAGS])
 fi
 
-if test -z "$LLVM_CONFIG"; then
-    if test -n "$llvm_prefix"; then
-        AC_PATH_TOOL([LLVM_CONFIG], [llvm-config], [no], ["$llvm_prefix/bin"])
-    else
-        AC_PATH_TOOL([LLVM_CONFIG], [llvm-config], [no])
-    fi
-fi
+#if test -z "$LLVM_CONFIG"; then
+    #if test -n "$llvm_prefix"; then
+        #AC_PATH_TOOL([LLVM_CONFIG], [llvm-config], [no], ["$llvm_prefix/bin"])
+    #else
+        #AC_PATH_TOOL([LLVM_CONFIG], [llvm-config], [no])
+    #fi
+#fi
+LLVM_CONFIG=no
 
 llvm_add_component() {
     new_llvm_component=$1
-- 
1.9.1

